builtin/trap_osh.py

```diff
@@ -309,14 +309,7 @@ def _PrintNames(self):
             # EXIT is 0, but we hide that
             print('   %s' % hook_name)
 
-        # Iterate over signals and print them
-        n = signal_def.MaxSigNumber() + 1
-        for sig_num in xrange(n):
-
-            sig_name = signal_def.GetName(sig_num)
-            if sig_name is None:
-                continue
-            print('%2d %s' % (sig_num, sig_name))
+        signal_def.PrintSignals()
```

Comment:
As mentioned on Zulip, `signal_def.py` is not translated

So it could go in either `builtin/process_osh.py` I guess .... (there doesn't seem to a common library file in `builtin/`)